default.build

```diff
@@ -170,9 +170,12 @@
   <target name="build_windows10" description="Build Windows 10 UAP" depends="clean">
     <if test="${os == 'Win32NT'}">
       <if test="${file::exists('c:\Program Files (x86)\MSBuild\Microsoft\WindowsXaml\v14.0\Microsoft.Windows.UI.Xaml.CSharp.targets')}">
-        <exec program="Protobuild" commandline="-generate WindowsUAP" />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU"' />
-        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUAP.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU"' />
+        <exec program="Protobuild" commandline="-generate WindowsUniversal" />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Clean /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
+        <exec program="${msbuild14dir}\msbuild.exe " commandline='MonoGame.Framework.WindowsUniversal.sln /t:Build /p:Configuration=Release /p:Platform="Any CPU" /p:ResolveNuGetPackages=false' />
```

Comment:
> Okay, that is not the maintenance structure that I believed was active. As far as I knew, anyone with the "Owner" tag on this repository was a maintainer, and that the decisions were made either collectively or ad-hoc based on who decides to merge things in.

There are a few others that used to be active maintainers (moved on or are too busy now) that still show up as owners.  Like you will see @dellis1972 merge some simple PRs here and there when he finds some time.  And you'll see us ask other owners and even non-owners for opinions when we're unsure about an issue.

Still you will see that most all PRs merged are handled by @KonajuGames or I and all big changes see us agree before things are merged.

This isn't new... this is how things have been running for almost 2 years now.

> but if those people aren't responsible or a part of decision-making

Everyone that wants to chime in here... for good or bad... participates in the decision making process.  We don't just ignore the discussion here and do what we please...  we consider everything said and make a decision which the current maintainers and previous maintainers that care to participate agree to.